src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -308,7 +308,8 @@ protected function getCategoryListRepresentation(Request $request, $parentKey =
         $listBuilder->addSelectField($fieldDescriptors['locale']);
         $listBuilder->addSelectField($fieldDescriptors['defaultLocale']);
 
-        $listBuilder->addGroupBy($fieldDescriptors['id']);
+        // I think this "group by" is not required
+        //$listBuilder->addGroupBy($fieldDescriptors['id']);
```

Comment:
That was probably because of conflicts... In that case you would have to rebase your branch, so that it is mergable. This is also described in our [documentation](http://docs.sulu.io/en/latest/developer/contributing/pull-requests.html).

Usually we try to do atomic commits for such small fixes. I can do that for you now, just that you know it for the next time :-)